### DISCUSSION TITLE
How to turn off IdeaVim keybinds like `control+f` `control+e`

### DISCUSSION THREAD

**Discussion by nikivdev** [JetBrains/ideavim#873] [General]
They are breaking these shortcuts for me:

<img width="1319" alt="image" src="https://github.com/JetBrains/ideavim/assets/6391776/0ab470ed-22fb-4624-8008-193efd4afed9">

Control + F won't work with IdeaVim on.

Thank you.

**Comment by OptimusCrime** [comment]
If you go to IdeaVim settings, are you sure that Control+F etc is mapped with IDE as the handler?

![Skärmavbild 2024-05-08 kl  15 33 47](https://github.com/JetBrains/ideavim/assets/2326278/93884950-40cc-4aae-a217-715ead39bb05)

**Reply by nikivdev** [reply]
if i turn off ideavim it works

**Reply by nikivdev** [reply]
where can I find IdeaVim settings?

<img width="1728" alt="image" src="https://github.com/JetBrains/ideavim/assets/6391776/7ad30722-4bdc-4bf0-9b8c-d5376386d0a1">

I only see this

**Reply by nikivdev** [reply]
what I see for `IdeaVim` keymap

<img width="1342" alt="image" src="https://github.com/JetBrains/ideavim/assets/6391776/7099fac2-3b22-4399-8094-56c73040b957">

I don't see where `ctrl+f`, `ctrl+e` is defined but it 100% overwrites my keymaps I set for the editor itself.

**Reply by nikivdev** [reply]
@OptimusCrime 

I even tried this:

<img width="307" alt="image" src="https://github.com/JetBrains/ideavim/assets/6391776/ebbd0e9c-218e-4231-ab6f-d185fce43a43">

In theory it should disable those keybinds but still, its overwriting my regular keybinds.

**Reply by nikivdev** [reply]
ok fixed it somehow, this started showing up
<img width="552" alt="image" src="https://github.com/JetBrains/ideavim/assets/6391776/d6fe848d-90fb-4715-b692-707c9a1251e8">

it didn't before

**Reply by citizenmatt** [reply]
These shortcuts aren't maps, but standard Vim shortcuts, so can't be unmapped. You can specify to treat the shortcut as an IDE or Vim shortcut though, as mentioned by @OptimusCrime above. You need to go to the _Editor | Vim_ settings page, and there you can choose how it's processed. 

The notification popup you're seeing controls this setting - the first time you use a Vim shortcut, it tells you that it's a Vim shortcut and allows you to switch and use it as an IDE shortcut. Clicking Configure will take you to the _Editor | Vim_ settings page.
